frontend/public/locales/en/translation.json

```diff
@@ -163,7 +163,10 @@
     "nbContributors_other": "{{count}} contributors",
     "criteriaRatedHigh": "Rated high:",
     "criteriaRatedLow": "Rated low:",
-    "labelShowSettings": "Show settings related to this video"
+    "labelShowSettings": "Show settings related to this video",
+    "unsafeNotEnoughContributor": "May be good but we have low confidence as this has been rated by only one contributor",
```

Comment:
How about not hard coding the number `one` in the translation?

The unsafe characteristic of a video can evolve if the Tournesol community grows, and in the future an unsafe video could be a video with two distinct contributors, three, or even more.

**edit** I now think the original text is better than my following proposition, as I found "low confidence" better than "uncertain", because there is always a bit of uncertainty in a video relevance, even if its score is high. But I'll leave the comment as is, for this history.

Here is an idea:

```javascript
"The relevance of this score is uncertain, due to a low number of distinct contributors."
```

or even

```javascript
"The relevance of this video's score is uncertain, due to a low number of distinct contributors."
```

It also avoid using the pronoun `we`. Who is `we`? The Tournesol community? The Tournesol agorithm? Even if we can agree on an answer, I think we ( the Tournesol team : ) can be more clear by avoiding `we`.

What do you think @amatissart @lfaucon @farnots ?

Reply:
I strongly like avoiding "we". I slightly prefer "uncertain" than "low confidence" but like both options

Reply:
Done I updated to avoid `we`